test/error_strings_custom_functions_test.go

```diff
@@ -13,3 +13,10 @@ func TestErrorStringsWithCustomFunctions(t *testing.T) {
 		Arguments: args,
 	})
 }
+
+func TestErrorStringsIssue1243(t *testing.T) {
+	args := []any{"errors.New", "errors.Wrap"}
```

Comment:
Should it be this?

```suggestion
	args := []any{"errors.Wrap"}
```

You are testing errors.New that is in the default one. And the issue is about making sure adding wrap doesn't invalidate existing ones